Development

mapra99 opened the pull request:
Requesting Second code review. The first code review can be found on the pull request from the branch named "Main-branch". We created this new branch with a more adequate name, following the suggestion made by the first reviewer.

matheus-fls committed:
Started header and added a logo

matheus-fls committed:
Added logo images to the Header and started Forms section

matheus-fls committed:
Added form fields

mapra99 committed:
Button created

mapra99 committed:
CSS File created

matheus-fls committed:
Positioned content and added some styling

mapra99 committed:
Title and text from form header centralized, and some style added

mapra99 committed:
Style added to border and text

matheus-fls committed:
Changed background color and input field positioning

mapra99 committed:
Images without background

matheus-fls committed:
Modified input styling

mapra99 committed:
Button Style added. Placeholders implemented and resized anchorage

matheus-fls committed:
Linted and organized HTML

matheus-fls committed:
Ordered properties alphabetically

matheus-fls committed:
Update README.md

matheus-fls committed:
Update README.md

Corrected bug

matheus-fls committed:
Update README.md

mapra99 committed:
Merge branch 'master' into main-branch

mapra99 committed:
Added Footer

matheus-fls committed:
Divided services logo into 3 different logos

mapra99 committed:
Added three icons separately, one for each service

matheus-fls committed:
Organized CSS alphabetically and repositioned header

mapra99 committed:
Added a pattern validation for Telephone entry

mapra99 committed:
Fixed bugs from Linter

mapra99 committed:
Fixing more bugs thrown by linter

mapra99 committed:
Merge pull request #2 from mapra99/main-branch

Main branch

matheus-fls committed:
Fixed HTML structure

matheus-fls committed:
Fixed small error on line 4

matheus-fls committed:
Changed section to div container

Ispirett commented:
Really Awesome work folks :1st_place_medal:  You are 95% there. Final changes
- [x] Add the section under the Create Account button that starts with **By clicking Create account, you agree to our** and this section **Invisible reCAPTCHA by Google Privacy Policy and Terms of Use.**

When finished please submit for another review request.

matheus-fls committed:
Removed csslint due to issues

For more information on that there check out the link below:
https://github.com/microverseinc/linters-config/pull/1#issue-296583961

matheus-fls committed:
Added privacy statement text below submit button

And some style

mapra99 committed:
Added Captcha Privacy Policy text in the footer of fieldset

mapra99 commented:
Added Terms of Use section below the submit button and in the footer of the fieldset.

bolah2009 commented on index.html:
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
```
`Awesome`

Nice job using `lang="en"` (or variant) to the `html` start tag, this is good for internationalisation. You should check this [link for more info](https://www.w3.org/International/questions/qa-html-language-declarations#basics).  :+1: :star2:

bolah2009 commented on index.html:
```diff
@@ -0,0 +1,110 @@
+    <main>
+
+        <section>
+            <form name="create-account">
```
`AWESOME` 

Nice job using `form` element, this makes the section represent a document section that contains interactive controls for submitting information to a web server. :+1:

bolah2009 commented on index.html:
```diff
@@ -0,0 +1,110 @@
+            <div class="services-container center-div">
+                <a href="#" class="serviceIcon"><img src="images/turboIcon.png" alt="Turbotax Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/quickIcon.png" alt="Quickbooks Logo"></a>
+                <a href="#" class="serviceIcon"><img src="images/mintIcon.png" alt="Mint Logo"></a>
```
`AWESOME`

Nice job, :clap: :star2:  adding an `alt` attribute to the `<img>` element, this is a very important accessibility attribute.
`Examples of scenarios where users benefit from text alternatives for images` are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

bolah2009 commented on index.html:
```diff
@@ -0,0 +1,110 @@
+                        Email address
+                    </label>
+                    <div class="input-cont">
+                        <input type="email" name="email" id="email" placeholder="You'll verify this later" required>
```
`AWESOME`

Good job associating the `<label>` with an `<input>` element. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```


You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

bolah2009 reviewed (approved):
# Congratulations Team! :tada:

You've done an awesome Job :muscle:

I'm impressed and proud of your progress, you should be too :clap:

Good luck with your next project
Keep it up :star2:

Cheers
Bola

mapra99 commented:
Review approved. Merging to master
